sdcm/nemesis.py

```diff
@@ -1474,6 +1474,25 @@ def target_node_reported_un_by_others():
         assert self.tester.hints_sending_in_progress() is False, "Hints are sent too slow"
         self.tester.verify_no_drops_and_errors(starting_from=start_time)
 
+    def _validate_snapshot(self, nodetool_cmd, snapshot_content):
```

Comment:
1) add type hints
2) add docstring that explains how this method validates snapshot

Reply:
Done